docs/content/docs/test/Runner.md

```diff
@@ -527,7 +527,7 @@ instrumentation {
 
 ## Find out how filters were applied
 
-If build finished successfully there will be files with debug information
+If build finishes successfully It will produce files with debug information
```

Comment:
```suggestion
If build finishes successfully, it will produce files with debug information
```
Is it one sentence?

Reply:
It is First conditional

Reply:
I see. This is why I've proposed to use ", it"

https://www.ef.com/wwen/english-resources/english-grammar/type-1-conditional/